server/controller/insertQuestions.js

```diff
@@ -0,0 +1,23 @@
+const insertQuestionsQuery = require('../database/queries/insertQuestionsQuery');
+
+const insertQuestions = (req, res) => {
+	const questions = req.body;
```

Comment:
You should check the question,"empty and validate"...
 checking will protect your server